src/BlazorUI/Bit.BlazorUI/Components/Buttons/MenuButton/BitMenuButton.scss

```
@@ -104,18 +107,25 @@
 }
 
 .bit-mnb-cal {
+    --bit-mnb-cal-opa: 0;
+    --bit-mnb-cal-tra: translateY(#{spacing(-1.25)});
     display: none;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)